deprecate.dd

```diff
@@ -713,7 +713,7 @@ $(H4 Rationale)
 
 
 
-$(H3 $(DEPNAME if (v; e)))
+$(H3 $(DEPNAME if (v; e, variable-delcaration-in-if-without-auto)))
```

Comment:
Misplaced closing parenthesis here. `if (v; e` needs to be closed before `, variable-delcaration-in-if-without-auto`.

Also a typo: de**lc**aration. Needs to be fixed in the link, too.